src/applications/ivc-champva/shared/components/applicantLists/ApplicantRelationshipPage.jsx

```diff
@@ -29,15 +29,6 @@ export function appRelBoilerplate({ data, pagePerItemIndex }) {
   const personTitle = 'Sponsor';
   const applicant = applicantWording(currentListItem, undefined, false);
 
-  // TODO: remove useFirstPerson when we're sure we won't want the functionality
-  // // Determine what tense/person the phrasing should be in
-  // const useFirstPerson =
-  //   data?.certifierRole === 'applicant' && +pagePerItemIndex === 0;
-  const useFirstPerson = false;
-
-  const relative = `${useFirstPerson ? 'I' : applicant}`;
-  const beingVerbPresent = useFirstPerson ? 'am' : 'is';
```

Comment:
This logic will always produce the same output because useFirstPerson is hardcoded to false. Removed this and set sensible values in the return statement of this function instead of keeping this bloat.